products/yarn.md

```diff
@@ -36,8 +36,8 @@ releases:
 
 -   releaseCycle: "1"
     releaseDate: 2017-09-05
-    eol: false
-    latestReleaseDate: 2022-06-08
+    eol: 2022-05-10
```

Comment:
Given there was no official EOL announcement we could use the date from https://github.com/yarnpkg/yarn/commit/fbcdb79a9878ef2d6e10e70c97adece2d94fa754, WDYT ?

In any case the product description also needs an update as it still says that yarn 1.x "will eventually reach end-of-life".

Reply:
Yeah, the current status seems to be "if there is a security issue in 1.x, a new release will happen somehow".

I wouldn't call it EOL, just not actively developed.

Reply:
@captn3m0 in the about section of https://github.com/yarnpkg/yarn it will divert even bug fixes to v2+, so I would say it is not maintained.

Reply:
@marcwrobel Sounds good we can use that date.

Reply:
@captn3m0, as of now you can see on https://github.com/yarnpkg/yarn description:

>  this (...repository...) being mostly kept for historical purposes and the occasional hotfix we publish to make the migration from 1.x to later releases easier.

It does not say anything about "if there is a security issue in 1.x, a new release will happen somehow". The security support was only available when Yarn 1.x was in maintenance mode (that's what I understood reading https://dev.to/arcanis/introducing-yarn-2-4eh1#what-will-happen-to-the-legacy-codebase). 

That's why I think @smeng9 is right and we should mark this as EOL. WDYT ?

Reply:
@arcanis Sorry to pull you in here. I understand you have spend a lot of efforts to push users to use yarn 2+, is yarn 1 eol now?

Reply:
If there's a security issue or Node.js incompatibility we'll attempt to release a hotfix (basically keeping the lights on).

Regular features/bugfixes however only happen on modern releases.